docs/messaging/gcp-pubsub.md

```diff
@@ -50,11 +64,73 @@ flowchart LR;
 | Links |  |  | Span Create A, Span Create B |
 | SpanKind | `PRODUCER` | `PRODUCER` | `CLIENT` |
 | Status | `Ok` | `Ok` | `Ok` |
+| `gcp.project_id` | `"P"` | `"P"` | `"P"` |
 | `messaging.batch.message_count` |  |  | 2 |
-| `messaging.destination.name` | `"T"` | `"T"` | `"T"` |
+| `messaging.destination.name` | `"projects/P/topics/T"` | `"projects/P/topics/T"` | `"projects/P/topics/T"` |
 | `messaging.operation` | `"create"` | `"create"` | `"publish"` |
 | `messaging.message.id` | `"a1"` | `"a2"` | |
 | `messaging.message.envelope.size` | `1` | `1` | |
 | `messaging.system` | `"gcp_pubsub"` | `"gcp_pubsub"` | `"gcp_pubsub"` |
 
+### Unary Pull Example
+
+```mermaid
+flowchart TD;
+  subgraph CONSUMER
+  direction LR
+  R1[Receive m1]
+  SM1[Ack m1]
+  EM1[Modack m1]
+  end
+  subgraph PRODUCER
+  direction LR
+  CM1[Create m1]
+  PM1[Publish]
+  end
+  %% Link 0
+  CM1-. link .-PM1;
+  %% Link 1
+  CM1-. link .-R1;
+  %% Link 2
+  R1-. link .-SM1;
+  %% Link 3
+  R1-. link .-EM1;
+
+  %% Style the node and corresponding link
+  %% Producer links and nodes
+  classDef producer fill:green
+  class PM1,CM1 producer
+  linkStyle 0 color:green,stroke:green
+
+  %% Consumer links and nodes
+  classDef consumer fill:#49fcdc
+  class R1 consumer
+  linkStyle 1 color:#49fcdc,stroke:#49fcdc
+
+  classDef ack fill:#577eb5
+  class SM1 ack
+  linkStyle 2 color:#577eb5,stroke:#577eb5
+
+  classDef modack fill:#0560f2
+  class EM1 modack
+  linkStyle 3 color:#0560f2,stroke:#0560f2
+```
+
+| Field or Attribute | Span Create A | Span Publish A | Span Receive A | Span Modack A | Span Ack A |
+|-|-|-|-|-|-|
+| Span name | `T create` | `publish` |  `S receive` | `S modack` |`S ack` |
+| Parent |  |  |  | |  |
+| Links |  | Span Create A | Span Create A | Span Receive A | Span Receive A |
+| SpanKind | `PRODUCER` | `PRODUCER` | `CONSUMER` |`CLIENT` |`CLIENT` |
+| Status | `Ok` | `Ok` | `Ok` |`Ok` | `Ok` |
+| `gcp.project_id` | `"P"` | `"P"` | `"P"` |  `"P"` |  `"P"` |
+| `messaging.destination.name` | `"T"`| `"T"`| `"S"` | `"S"` |`"S"` |
```

Comment:
No, instead of using the full resource name, we are splitting it into two attributes, one for the topic/subscription and one for the project id.

I had a similar question, but there is no attribute for the messaging.source at the moment. This might change in the future